epimodel/exports/epidemics_org.py

```diff
@@ -334,6 +353,8 @@ def process_export(args) -> None:
         timezone, index_col="Code", keep_default_na=False
     )
 
+    un_age_dist_df: pd.DataFrame = pd.read_csv(un_age_dist, index_col="Code M49")
```

Comment:
I know you wanted M49 (and I would be fine with that) but everything else was ISO Alpha 2 so I just tried to follow that. I.e. consistency matters more to me than the chosen standard.

Reply:
1. I just quickly needed a source of age data and this was at hand already.
2. Its not quite the same because 'Code' is our internal thing which means `Iso2` on the country level. 
3. `M49` has codes for more things (larger regions than just country). 

Taking 2+3 I think we should just take what is in the data and not try to artificially force everything to use `Code`. But per 1 I don't really mind either way and this was just a quick fix.

Reply:
OK, let's keep it as you have it.